### ISSUE TITLE
the "benchmarks" section adds little to the paper

### ISSUE THREAD

**Issue by jowens** [eunaeuna/GraphFrameworkOnGPUs#4]
Spending a page on what these benchmarks are is not a good use of space. They have been described in dozens of papers before, including papers that just about every one of the authors have written. It would perhaps be useful to compare different GPU approaches to solve these problems—how these problems map to the GPU—but as is, it's just a generic "here's what this benchmark is".

"Why we picked these" (the first paragraph) is useful.

**Comment by eunaeuna** [comment]
I agree. I have to say we used to spend more pages to explain the basics and cut them off mostly. We can make it shorter if we exceed the space limit 10 pages.

> It would perhaps be useful to compare different GPU approaches to solve these problems
I was personally very impressed by the counterpart in Gunrock paper; succinct and informative.
We thought the audience of this paper is more general.. Also I cited Gunrock paper which includes useful description for those...

Let me think about your suggestion. : )
